lms/static/sass/shared/_course_object.scss

```diff
@@ -180,78 +191,116 @@
         }
       // }
 
-      .info {
-        background: $content-wrapper-bg;
-        height: 220px + 130px;
-        left: 0px;
-        position: absolute;
-        top: 0px;
-        @include transition(all 0.15s linear 0s);
-        width: 100%;
-        overflow: hidden;
-
+      .course-image {
         .cover-image {
-          height: 200px;
+          height: 160px;
           overflow: hidden;
-          width: 100%;
+
+          &:before {
+            @include fade-transition();
+            content: '';
+            position: absolute;
+            top: 0;
+            left: 0;
+            background: black;
+            opacity: 0;
+            width: 100%;
+            height: 160px;
+            z-index: 10;
+          }
+
+          &:after {
+            @include fade-transition();
+            @include box-sizing(border-box);
+            @include calc( top, '80px - 25px');
+            @include calc( left, '50% - 100px');
+            content: 'LEARN MORE';
```

Comment:
I don't think this type of content should be rendered by the CSS layer or things. If its a call to action, we should make sure its:

* accessible (adding @cptvitamin to chime in on this angle)
* i8tn-friendly (adding @sarina to chime in on this angle as well)

I'd prefer to have this in the DOM and to have the CSS target a UI-based class name to show/hide/transition its visual display.